src/components/RightSidebar/RightSidebar.vue

```diff
@@ -391,4 +484,23 @@ export default {
 	height: 100%;
 }
 
+.chat-button__unread-messages-counter {
+	position: absolute;
+	bottom: 2px;
+	right: 2px;
+	pointer-events: none;
+	color: var(--color-primary-element);
```

Comment:
On stable29 | With a patch
-- | --
![image](https://github.com/nextcloud/spreed/assets/93392545/6de4d757-4e26-47c6-854b-21794cd09d99) | ![image](https://github.com/nextcloud/spreed/assets/93392545/79338995-3df3-4492-8f44-bdc3be6cc6ec)

```suggestion

	&.counter-bubble__counter--highlighted {
		color: var(--color-primary-text);
	}
```

Reply:
This hasn't changed since Feb.2023 https://github.com/nextcloud/spreed/commit/d3a062e25ea770dc62df4957beb58f487fe3fb98, and not related to this PR changes, so could be fixed in follow-up